src/target/riscv/batch.h

```diff
@@ -58,14 +58,14 @@ bool riscv_batch_full(struct riscv_batch *batch);
 /* Executes this scan batch. */
 int riscv_batch_run(struct riscv_batch *batch);
 
-/* Adds a DMI write to this batch. */
-void riscv_batch_add_dmi_write(struct riscv_batch *batch, unsigned int address, uint64_t data,
+/* Adds a DM write to this batch. */
```

Comment:
Detail, for clarity:

```suggestion
/* Adds a DM register write to this batch. */
```